media/css/externalpages/pocket/components/_add.scss

```diff
@@ -30,49 +41,36 @@ $image-path: '/media/img/externalpages/pocket';
 }
 
 .add-header {
-    &__wrapper {
-        margin: 0 auto;
-        max-width: 1200px;
-        padding: $spacing100;
-        width: 70%;
-    }
-
-    &__context {
-        display: flex;
-        &-image {
-            width: 60%;
-
-            img {
-                display: block;
-                height: 100%;
-                width: 100%;
-            }
-        }
-
-        &-info {
-            display: flex;
-            align-items: flex-start;
-            flex-direction: column;
-            justify-content: space-around;
-            margin-left: 30px;
-            width: 35%;
+    .mzp-c-button {
+        background-color: $color-button-red;
+        border: 2px solid $color-button-red;
+
+        &:hover,
+        &:focus,
+        &:active {
+            text-decoration: underline!important;
+            background-color: $color-button-red;
```

Comment:
Had to copy the styles from the original state to override Protocol `mzp-c-button` styling on the pseudo states.

Reply:
I think there should be a space between `underline` and `!important`